### DISCUSSION TITLE
通过nonebot脚手架创建的项目启动不起来

### DISCUSSION THREAD

**Discussion by jdfcc** [nonebot/discussions#431] [问答] [answered]
本人纯菜鸡，刚接触nonebot,按照nonebot文档 [快速上手](https://nb2.baka.icu/docs/quick-start)创建完项目后使用```nb run --reload```命令启动项目后就没反应了。
python版本为```3.10.7```
这是运行命令后的截图
![image](https://github.com/nonebot/discussions/assets/75058546/ed8097eb-1163-4192-95f5-9ee7f8e9da99)
这是安装配置
![image](https://github.com/nonebot/discussions/assets/75058546/32da0140-2761-4daf-a83f-8d7ace0708ac)

**Comment by yanyongyu** [comment]
换一个终端试一下运行？看上去没启动TUI

**Reply by jdfcc** [reply]
使用管理员启powershell和cmd运行项目都启不起来🥲

**Reply by yanyongyu** [reply]
我这都是正常的，要不你试下windows terminal能不能启动？

**Reply by yanyongyu** [reply]
启动后什么键都别按看下会不会退出

**Reply by jdfcc** [reply]
还是不行，启动到差不多一半的时候终端会刷新一下然后就退出了

**Comment by RF-Tar-Railt** [comment, marked as answer]
去掉fastapi试试，就直接用None驱动器

**Comment by jdfcc** [comment]
![闪退截图](https://github.com/nonebot/discussions/assets/75058546/eb656c53-b68f-4f88-b222-628b25c573fe)

**Reply by RF-Tar-Railt** [reply]
日志开到debug看看？

**Reply by RF-Tar-Railt** [reply]
环境配置LOG_LEVEL=DEBUG

**Comment by jdfcc** [comment]
注释掉```.env.prod```文件里面的```DRIVER=~fastapi```后成功运行😽
